omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -806,38 +818,6 @@ private void addSubclassOperations() {
 		}
 	}
 	
-	@Deprecated
-	private List<org.openmrs.module.webservices.docs.swagger.Parameter> getParametersListForSearchHandlers(
-	        String resourceName, String searchHandlerId, int queryIndex) {
-		List<org.openmrs.module.webservices.docs.swagger.Parameter> parameters = new ArrayList<org.openmrs.module.webservices.docs.swagger.Parameter>();
-		String resourceURL = getResourceUrl(getBaseUrl(), resourceName);
-		for (SearchHandlerDoc searchDoc : searchHandlerDocs) {
```

Comment:
Is removing this block of code related to? "Correct host property in OpenAPI Spec"

Reply:
See the comment above.